conmech/state/body_position.py

```diff
@@ -97,35 +92,45 @@ def element_volume_part_numba(face_nodes):
     return volume / nodes_count
 
 
-class BodyPosition:
+class BodyPosition(Mesh):
```

Comment:
This inheritance was break on purpose: https://github.com/KOS-UJ/conmech/pull/76/commits/a5b44d2e51cbc2c89fff27490cfc94d5de4db39f
There is no reason to keep it.

Reply:
Inheritance changed to composition